packages/ng/popup-employee/card/panel/user-popover-panel.component.ts

```diff
@@ -17,7 +17,7 @@ import { InjectParameterPipe } from '../pipe/inject-parameter.pipe';
 import { isFutureOrTodayPipe, IsFuturePipe } from '../pipe/is-future.pipe';
 import { LeaveEndsDisplayPipe } from '../pipe/leave-ends-display.pipe';
 import { ILuUserPopoverPanel } from './user-popover-panel.model';
-import { catchError, map } from 'rxjs/operators';
+import { catchError, map, switchMap } from 'rxjs/operators';
```

Comment:
Shouldn't all imports be from rxjs now ? 🤔 https://rxjs.dev/guide/importing#new-in-rxjs-v720

Reply:
Indeed, idk if this PR should do that change because it's just a hotfix and refactor is already starting but maybe we should have a lint rule for this to fix it once for the entire LF repo?

Reply:
Agreed, out of scope for now but weird that it wasn't caught automatically.